backend/de.metas.cucumber/src/test/resources/de/metas/cucumber/features/invoice/invoicePaymentAllocation.feature

```diff
@@ -1876,13 +1879,13 @@ Feature: invoice payment allocation
       | customerCreditMemo | creditMemoProduct | 1 PCE       | 434.58 | tax19%   |
     And the invoice identified by customerCreditMemo is completed
     And metasfresh contains C_Payment
-      | Identifier     | C_BPartner_ID | PayAmt      | IsReceipt | C_BP_BankAccount_ID |
-      | inboundPayment | vendor1       | -434.97 EUR | true      | org_EUR_account     |
+      | Identifier     | C_BPartner_ID    | PayAmt      | IsReceipt | C_BP_BankAccount_ID |
+      | inboundPayment | serviceProvider1 | -434.97 EUR | true      | org_EUR_account     |
     And the payment identified by inboundPayment is completed
 
-    When allocate payments to invoices
+    When allocate payments to invoices with verbatim DiscountAmt and InvoiceProcessing.FeeAmt
```

Comment:
IMHO i would expect a REMADV specific step and not adding more complexity to current allocate payments.
Ofc, that includes setting up the invoice processing company as we do on real systems (and not mock it).

Reply:
thx for the advices. my first prio and what most worries my right now to acutally know how to do the accounting. After that i will gladly discuss step-names

Reply:
Next i'm going to implement dedicated cucumber for Remittance-Advices. 
Then I'm going to drop things that are obsolte and improve the rest.
Until that's done, this steptdef can serve as as a temporary solution to create the allocations for which your are going to extend Doc_AllocationHdr.